src/utils/getCertificate.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+function getCertificate(options) {
+  const key = fs.existsSync(options.key) && fs.readFileSync(options.key);
```

Comment:
Perhaps use async reads here?

Reply:
I can add that. I patterned its behavior after `generateCertificate` which uses sync calls. I'll update the PR to use `utils/fs.js` and `await`.